kubernetes/os_route_client.go

```diff
@@ -19,7 +16,7 @@ type OSRouteClient struct {
 
 // NewOSRouteClient creates a new client able to fetch OpenShift Routes API.
 func NewOSRouteClient() (*OSRouteClient, error) {
-	config, err := rest.InClusterConfig()
+	config, err := ConfigClient()
```

Comment:
I think that the whole os_route_client.go is not being used anymore.

Reply:
I removed old references to openshift routes